src/kube-runtime/package_cache.md

```diff
@@ -0,0 +1,52 @@
+## Package Cache
+
+Some runtime plugins use `apt update` and `apt install ...` to install packages. If the network connection from pai cluster to the apt repository is poor, `apt install` may take a long period of time or fail.
```

Comment:
Add notice for we only support `x64` package here?

Reply:
Done.